addons/sale/views/sale_views.xml

```diff
@@ -253,10 +266,26 @@
                                             <label for="product_uom_qty" string="Ordered Quantity"/>
                                             <div>
                                                 <field
-                                                    context="{'partner_id':parent.partner_id, 'quantity':product_uom_qty, 'pricelist':parent.pricelist_id, 'uom':product_uom, 'uom_qty_change':True, 'company_id': parent.company_id}"
-                                                    name="product_uom_qty" class="oe_inline"/>
-                                                <field name="product_uom" groups="uom.group_uom" class="oe_inline oe_no_button"
-                                                    attrs="{'readonly': [('state', 'in', ('sale','done', 'cancel'))]}"/>
+                                                    name="product_uom_qty"
```

Comment:
Seems like unnecessary diff. It is not easy to spot if there are changes. Same below, even if attrs have been modified so try to have only the attrs in diff.

Reply:
If we ever want to improve the code there need to be a point where we can actually change something.

This one I didn't change functionally so I will revert.

In the case below I did change the line, and I had to split it to understand it, so I feel it is enough to keep it this way.